js/trackers.json

```diff
@@ -0,0 +1,6 @@
+{
+    "trackers": [
```

Comment:
Might be better to make this a hash for faster loop up or even turn it into one big regex.